Scripts/Interfaces/Entity/IEditingContext.cs

```diff
@@ -12,6 +14,11 @@ public interface IEditingContext
         /// </summary>
         string name { get; }
 
+        /// <summary>
+        /// Settings denoting that the scene camera's (component_ values should be cloned on the XR runtime camera
+        /// </summary>
+        ContextSettings contextSettings { get; }
```

Comment:
Why this context settings class instead of individual settings items being exposed as interface members/requirements?

Reply:
I was initially trying to avoid adding the copySceneCameraSettings param to IEditingContext.  The initial number of settings were more numerous when accounting for PostFX settings (coming in a latter PR).  Though after I minimized the implementation, it now makes sense to merge it back into IEditingContext, and remove the ContextSettings struct (for now).